src/layers/Dialog/Dialog.tsx

```diff
@@ -64,6 +64,11 @@ export interface DialogProps extends Omit<GlobalOverlayProps, 'children'> {
    * Theme for the Dialog.
    */
   theme?: DialogTheme;
+
+  /**
+   * Custom motion props to override default animations.
+   */
+  motionProps?: MotionProps;
```

Comment:
Why not just extend the interface w/ these props so you don't have to pass an object? Seems more natural to do that imo.